driver-sync/src/test/functional/com/mongodb/client/unified/Entities.java

```diff
@@ -118,6 +119,7 @@ public final class Entities {
     private final Map<String, TestLoggingInterceptor> clientLoggingInterceptors = new HashMap<>();
     private final Map<String, TestConnectionPoolListener> clientConnectionPoolListeners = new HashMap<>();
     private final Map<String, TestServerListener> clientServerListeners = new HashMap<>();
+    private final Map<String, TestClusterListener> clientClusterListeners = new HashMap<>();
```

Comment:
The new code using `clientClusterListeners` in this PR was written based on the existing code that uses `clientServerListeners`.